cli/cmd/volume.go

```diff
@@ -202,13 +210,36 @@ var volumeGetCmd = &cobra.Command{
 	},
 }
 
+func volumeOptionJsonHandler(cmd *cobra.Command, volname string, options []string) error {
+	vopt := make(map[string]string)
+	for op,val := range options {
+		if op % 2 == 0 {
+			vopt[val] = options[op+1]
+		}else{
+			continue
```

Comment:
else statement not adding any value here, can be removed

Reply:
Right. ok. Noted.